src/pudl/analysis/timeseries_cleaning.py

```diff
@@ -1579,17 +1580,21 @@ class SimulateFlagsSettings:
     """Min ratio of bad points in a section of data to be used for reference."""
     max_flag_rate: float = 0.5
     """Max ratio of bad points in a section of data to be used for reference."""
+    output_io_manager_key: str = "io_manager"
```

Comment:
Allow the io-manager for the simulated asset to be configurable so we can output for visualization purposes.